hedera-mirror-importer/src/main/resources/application.yml

```
@@ -45,9 +45,9 @@ management:
     health:
       group:
         liveness:
-          include: ping
+          include: ping, streamFileActivity
         readiness:
-          include: db, diskSpace, ping, redis
+          include: db, diskSpace, ping, redis, streamFileActivity
```

won't this cause adverse effect because specifically for balance `streamFileActivity` the first `UP` may take quite some time?

Perhaps we should default balance `streamFileActivity` to `UP`

---

I think if we forced the default as `UP` it would be misleading.
However, it is true that at startup for balance it'll go from `UNKNOWN` to `DOWN` until the first file is parsed.
It's tough since the frequency is outside of the parser control and the lack of a count within the window isn't a bad thing.

My hesitation with the mean approach was that on the first counter the mean would be 0.

Thinking on this a bit more as I think the right course which is the intention here is to have it `UNKWOWN` until a file has been parsed

---

my concern is the pod might get restarted if it takes up to 15 minutes for the balance `streamFileActivity` to be `UP`

---

I addressed the edge case by adding logic to continue to return the cached result  between startTime + first fileClose interval.

Good point on the pod restart, @steven-sheehy what do you think?

---

Based off of offline discussion this should be good, will do some local verification before merging

---

Verified locally, highlighted a missing case which I updated in the recent commit. Should be all set now